pkg/startup/startup_windows.go

```diff
@@ -96,3 +96,7 @@ func ensureNetworkForOS(ctx context.Context, c client.Interface, nodeName string
 	logrus.Info("Ensure network is done.")
 	return nil
 }
+
+// Placeholder to compile windows binary.
+func MonitorIPAddressSubnets() {
```

Comment:
I think this implementation is wrong for Windows - rather than exit immediately, we need this function to "sleep" forever.

You could have it trying to a read from a channel with no producers, for example.

Reply:
I was thinking, for lake of monitoring IP address/subnet in Windows, to retain the old behavior which is similar to `run-once`. I have 2 questions --

1. what's the advantage of 'sleep' for Windows when we are not monitoring the IP?
2. should we make this -- autodetect based on time alone for Windows?

Reply:
We could do the second option, yeah.

The main thing is that if we exit, then the service will have "crashed" and runsv will attempt to restart the service, which of course will then exit immediately.

Reply:
I will add auto-detect based on time for Windows.